sysutils/tanka/Portfile

```diff
@@ -3,9 +3,9 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/grafana/tanka 0.19.0 v
-# Delete this on next update to use golang PortGroup's default ('archive')
-github.tarball_from tarball
+go.offline_build    no
```

Comment:
as in the other PR, I am not convinced this is MacPorts' recommended/preferred approach (@herbygillot ?)